### ISSUE TITLE
Robot reporter support for Github enterprise server

### ISSUE THREAD

**Issue by silpagithubprivate** [joonvena/robotframework-reporter-action#19]
Hi,

Does this report supports GitHub Enterprise server?

We have tried using this, but the api call failed.
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/commits/comments#list-commit-comments"
}

As per the readme doc, it is possible to show the report to summary. In that case, how should we disable sha and pull_request_id?

**Comment by Rob-Bos** [comment]
Seems like the action uses the Docker container from [here ](https://github.com/joonvena/Robot-Reporter/blob/8ba8c4d4c247abf3722ab4c15a6e5ac4fc4401c1/main.go#L195)and that tries to post back a comment to the issue, but does so hardcoded to `api.github.com` instead of the server url:
```
Post "https://api.github.com/repos/owner/reponame/commits/commit-hash/comments": dial tcp ip-address: i/o timeout
```

Seems like the `GITHUB_API_URL` environment variable is not used correctly?

**Comment by rajbos** [comment]
Seems like the action uses the Docker container from [here ](https://github.com/joonvena/Robot-Reporter/blob/8ba8c4d4c247abf3722ab4c15a6e5ac4fc4401c1/main.go#L195)and that tries to post back a comment to the issue, but does so hardcoded to `api.github.com` instead of the server url:
```
Post "https://api.github.com/repos/owner/reponame/commits/commit-hash/comments": dial tcp ip-address: i/o timeout
```

Seems like the `GITHUB_API_URL` environment variable is not used correctly?

**Comment by joonvena** [comment]
Hi, yes this is correct. It initializes the Github client with default settings which then will default to using the `api.github.com`. I will make try to have time to fix it this week so it would actually use the `GITHUB_API_URL` variable value there for the `BaseURL`. Here is the part in the reporter code if interested:

https://github.com/joonvena/Robot-Reporter/blob/master/main.go#L195

**Comment by joonvena** [comment]
Hello @silpagithubprivate this is now fixed with v2.2 🙂
